Sources/ProjectAutomation/TargetDependency.swift

```diff
@@ -19,6 +19,7 @@ public enum TargetDependency: Equatable, Hashable, Codable, Sendable {
     case xcframework(path: String, status: LinkingStatus)
     case library(path: String, publicHeaders: String, swiftModuleMap: String?)
     case package(product: String)
```

Comment:
Since `package` already represents `runtime` dependency what about reusing that one using default values to make it backward-compatible?

```bash
case package(product: String, embedded: Bool = false)
```

Reply:
Thank you for review! I've updated this case to use associated value.